test/e2e/tests/send-eth.spec.js

```diff
@@ -426,10 +426,9 @@ describe('Send ETH from inside MetaMask to a Multisig Address', function () {
         await inputAmount.fill('1');
 
         // Continue to next screen
-        await driver.findClickableElement({ text: 'Next', tag: 'button' });
+        await driver.delay(regularDelayMs); // wait for Next button is enabled
```

Comment:
Why is this `delay` needed? Was `findClickableElement` timing out?

Reply:
In terms of removing old `findClickableElement, is because in `await driver.clickElement({ text: 'Next', tag: 'button' });` we have `findClickableElement` as part of that function. 

And the reason for adding extra delay here, is we have 10s time out for `clickElement`, and sometimes it was not enough for button to be enabled. One thing we can do is to pass a new timeout inside `clickElement` to override original `this.timeout`.

```
  async clickElement(rawLocator) {
    const locator = this.buildLocator(rawLocator);
    const element = await this.findClickableElement(locator); 
    await element.click();
  }
```

Reply:
These CI machines aren't _that_ slow; 10  seconds should be ample time for the form validation to run. Something doesn't seem right here.

Reply:
Makes sense, Gonna remove the delays completely and run a few times to see if this is the real problem 🙏